### PR TITLE
chore(buildkite): install aws cli v2 as system dependency

### PR DESCRIPTION
**Pull request by rexledesma**
## Summary
Required for `aws sso login`. 

## Test Plan
```
dagster-image build-all --name buildkite-integration-base --dagster-version 0.13.14
dagster-image build-all --name buildkite-integration --dagster-version 0.13.14
dagster-image push-all --name buildkite-integration-base
dagster-image push-all --name buildkite-integration
```

See buildkite tests succeed.

### CODE REVIEW

**Comment by rexledesma** [comment]
Current dependencies on/for this PR:
* master
  * **PR #6242**  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: python_modules/automation/automation/docker/images/buildkite-integration-base/Dockerfile [hunk 1]
```diff
@@ -159,3 +159,8 @@ RUN apt-get update -yqq \
     && apt-get install -yqq --no-install-recommends \
     libxmlsec1-dev \
     pkg-config
+
+# Install AWS CLI v2
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" \
```
Comment (jmsanders) [inline python_modules/automation/automation/docker/images/buildkite-integration-base/Dockerfile:164]: How did we use aws cli before (v1 or v2)? I know at least some jobs shelled out to it - were they managing their own installations?

**Review by jmsanders — APPROVED** [review]

**Comment by alangenfeld** [comment]
looking at the logs in buildkite it seems like its sourcing from `python_modules/automation/automation/docker/images/buildkite-integration/last_updated.yaml` which this didnt appear to update yet?

**Comment by alangenfeld** [comment]
ah ya you need to update the integration images that are built on top of the updated base

**Pull request closed without merging**